common/Cargo.toml

```diff
@@ -57,11 +57,12 @@ backend = [
     "axum/matched-path",
     "axum/json",
     "claims",
-    "hyper/client",
+    "hyper",
```

Comment:
Should this remove `hyper` instead?

Reply:
it's used in other places in backend feature